addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/radiora/handler/PhantomButtonHandler.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.lutron.internal.radiora.handler;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.lutron.LutronBindingConstants;
+import org.openhab.binding.lutron.internal.radiora.config.PhantomButtonConfig;
+import org.openhab.binding.lutron.internal.radiora.protocol.ButtonPressCommand;
+import org.openhab.binding.lutron.internal.radiora.protocol.LEDMapFeedback;
+import org.openhab.binding.lutron.internal.radiora.protocol.RadioRAFeedback;
+
+/**
+ * Handler for RadioRA Phantom buttons
+ *
+ * @author Jeff Lauterbach
+ *
+ */
+public class PhantomButtonHandler extends LutronHandler {
+
+    private PhantomButtonConfig config;
+
+    public PhantomButtonHandler(Thing thing) {
+        super(thing);
+        this.config = getConfigAs(PhantomButtonConfig.class);
```

Comment:
Can the config be updated in the lifetime of this object, I think so, then it will be disposed and re-initialized.